client/js/views/GameView.vue

```diff
@@ -1301,7 +1301,7 @@ export default {
         width: 100%;
         margin: 0 auto;
         overflow: auto;
-        font-size: 1.15em;
+        font-size: 1em;
```

Comment:
Lowered font to better fit the log. 
![image](https://user-images.githubusercontent.com/47369872/206310262-dfe0c354-b3ee-4275-9873-67d9d6f0138d.png)